src/js/utils/strings.js

```diff
@@ -112,13 +106,9 @@ export function offsetToSeconds(offset, duration, frameRate) {
 }
 
 export function prefix(arr, add) {
-    return map(arr, function(val) {
```

Comment:
Are you certain that `arr` is an `Array` or one of those HTML element arrays?

Reply:
Yep. This is only used in skins.js as follows:

```
addStyle([
    // controlbar text colors
    '.jw-controlbar .jw-icon-inline.jw-text',
    '.jw-title-primary',
    '.jw-title-secondary',
], 'color', config.text);

// "elements" is a bad name. This is an Array of strings (CSS classes)
// to prefix with the player id for specificity:
function addStyle(elements, attr, value, extendParent) {
    elements = prefix(elements, '#' + playerId + (extendParent ? '' : ' '));
```